### ISSUE TITLE
0.0.9 can break builds

### ISSUE THREAD

**Issue by domgraziano** [aquasecurity/trivy-action#28]
Using version 0.0.9 seems to be buggy 
```
    runs-on: ubuntu-latest
    steps:
      - name: Checkout code
        uses: actions/checkout@v2
      - name: Build an image from Dockerfile
        env:
          MY_TOKEN: ${{ secrets.MY_TOKEN }}
        run: |
          docker build . --tag this_repo
      - name: Run Trivy vulnerability scanner
        uses: aquasecurity/trivy-action@master
        with:
          image-ref: this_repo
          format: 'table'
          exit-code: '1'
          ignore-unfixed: true
          severity: 'CRITICAL,HIGH'
```

The `ignore-unfixed: true` will be ignored !

Tried the exact same with version 0.0.8 and it works (--ignore-unfixed is in the logs with that version)

**Comment by simar7** [comment]
@rahul2393 can you look into this?

**Comment by simar7** [comment]
Thanks for the report again @domgraziano – I've pushed a new release which should fix this issue. Please give it a try at https://github.com/aquasecurity/trivy-action/releases/tag/v0.0.10

**Comment by domgraziano** [comment]
yes confirming it works again

**Comment by domgraziano** [comment]
> Thanks for the report again @domgraziano – I've pushed a new release which should fix this issue. Please give it a try at https://github.com/aquasecurity/trivy-action/releases/tag/v0.0.10
> 
> Feel free to reopen this issue, if the problem still persists.

note that the version has been published as v0.0.10 instead of simply 0.0.10 as all the previous version this could cause quite a lot of confusion, I'd suggest to create a tag with 0.0.10 without deleting v0.0.10 as someone it might have used that already

I'd also suggest to delete 0.0.9 and recreate it from master (without caring about the fact that is identical to 0.0.10), that would automatically fix issues for other people trying to checkout 0.0.9 which holds the bug in it
